### DISCUSSION TITLE
blog/2024/06/17/boring-monday/

### DISCUSSION THREAD

**Discussion by giscus** [MingGH/996-ninja-giscus#15] [General]
# blog/2024/06/17/boring-monday/

无聊的周一，随便更新点内容敷衍一下

https://996.ninja/blog/2024/06/17/boring-monday/

**Comment by onepiecewyz** [comment]
你好，和你咨询一下，通过github的cloudflare-docker-proxy项目，部署我自己的docker代理后，国内访问worker依然是被墙，为什么使用你的docker.996.ninja没有被墙，你的worker是付费的么，还是多做了什么操作？

**Reply by MingGH** [reply]
我的works也是用的免费版本。
先排查一下是不是GFW的影响，能不能直接在服务器上通过curl请求通。
另外可以换几个其他的域名绑定works试试，可能被dns污染了，或者你的服务器的hosts文件dns映射错了
